lib/Makefile

```diff
@@ -212,7 +212,11 @@ ifeq ($(UNAME), Darwin)
   SHARED_EXT_VER = $(LIBVER).$(SHARED_EXT)
   SONAME_FLAGS = -install_name $(LIBDIR)/libzstd.$(SHARED_EXT_MAJOR) -compatibility_version $(LIBVER_MAJOR) -current_version $(LIBVER)
 else
-  SONAME_FLAGS = -Wl,-soname=libzstd.$(SHARED_EXT).$(LIBVER_MAJOR)
+  ifeq ($(UNAME), AIX)
+    SONAME_FLAGS =
```

Comment:
That part feels weird.
What happens to dynamic libraries in the context of AIX ?

Reply:
AIX linker doesn't simply understand -soname flag. GCC when calling the linker will pass the right arguments as long as "-shared" is provided. The "-o" flag is enough to have the correct soname.